include/Scrape.php

```diff
@@ -235,7 +235,9 @@ function scrape_feed($url) {
 	$a = get_app();
 
 	$ret = array();
-	$s = fetch_url($url);
+	$cookiejar = tempnam('/tmp', 'cookiejar-scrape-feed-');
```

Comment:
I think you should use [get_temppath()](https://github.com/friendica/friendica/blob/master/boot.php#L1888) function instead of hardwired value